ci/configure_bridge.sh

```diff
@@ -1,15 +1,25 @@
+#!/bin/bash
 
+echo "Disconnect old bridge"
+
+# Disconnect - may fail if not there
+sudo tedge disconnect c8y
+
+# From now on exit if a command exits with a non-zero status.
+# Commands above are allowed to fail
 set -e
 
 echo "Configuring Bridge"
 
+URL=$(echo $C8YURL | cut -c 9- - )
```

Comment:
I hope all our C8YURL values start with `https://` so that the range `9-` remains valid.

What's the meaning of that last independent `-` in cut command?

Reply:
If the https:// is missing we hopefully find out soon :) . 
The last dash only means to read from stdin.